README.md

```diff
@@ -2,7 +2,7 @@
 
 # ModelicaSpecification
 This repository contains the Modelica Language Specification, hosted at https://github.com/modelica/ModelicaSpecification.
-Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md)
+Development is organized within the [Modelica Association Project Language (MAP-LANG)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md).
```

Comment:
How about taking the opportunity to get the correct MAP-Lang casing?
```suggestion
Development is organized within the [Modelica Association Project Language (MAP-Lang)](https://modelica.org/projects) according to the [bylaws](https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md).
```

Reference: https://github.com/modelica/MAP-LANG_ProjectRules/blob/main/MAP-LANG-ProjectRules.md#abbreviated-name-of-project

Reply:
By the way, @sjoelund, here's another place where the all upper case LANG is wrong:
https://modelica.org/projects.html

Reply:
Yes. Did that - together renaming the project.

Reply:
Nice! Then https://modelica.org/projects.html is the only place I'm aware of that remains to be fixed.